### PR TITLE
[ZEPPELIN-1776] substitute null check for the refactored Revision class

### PR DESCRIPTION
**Pull request by khalidhuseynov**
### What is this PR for?
This is to fix minor bug introduced by #1697 

### What type of PR is it?
Bug Fix

### Todos
* [x] - substitute null check

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1776

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java [hunk 1]
```diff
@@ -1397,7 +1397,7 @@ private void checkpointNote(NotebookSocket conn, Notebook notebook,
     String commitMessage = (String) fromMessage.get("commitMessage");
     AuthenticationInfo subject = new AuthenticationInfo(fromMessage.principal);
     Revision revision = notebook.checkpointNote(noteId, commitMessage, subject);
-    if (revision != null) {
+    if (!Revision.EMPTY.equals(revision)) {
```
Comment (anthonycorbacho) [inline zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java]: maybe we can come up with a better implementation here like `revision.isEmpty()` or `Revision.isEmpty(revision)`

what do you think

Comment (khalidhuseynov) [inline zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java]: @anthonycorbacho yeah right, addressed in 13178aee18c0a4aed1dda0c72750e08c9ed9f53c

**Comment by khalidhuseynov** [comment]
@anthonycorbacho this is done, what do you think?

**Comment by anthonycorbacho** [comment]
LGTM thank for the contrib

**Comment by khalidhuseynov** [comment]
ping

**Comment by Leemoonsoo** [comment]
LGTM and merge to master if there're no further discussions.

**Pull request closed without merging**